packages/wdio-applitools-service/src/index.js

```diff
@@ -0,0 +1,73 @@
+import logger from 'wdio-logger'
+import { Eyes, Target } from '@applitools/eyes.webdriverio'
+
+const log = logger('wdio-applitools-service')
+
+const DEFAULT_VIEWPORT = {
+    width: 1440,
+    height: 900
+}
+
+export default class ApplitoolsService {
+    constructor () {
+        this.eyes = new Eyes()
+    }
+
+    /**
+     * set API key in onPrepare hook and start test
+     */
+    beforeSession (config) {
+        const key = process.env.APPLITOOLS_KEY || config.applitoolsKey
+        const applitoolsConfig = config.applitools || {}
+
+        if (!key) {
+            throw new Error(`Couldn't find an Applitools private key in config nor environment`)
+        }
+
+        this.isConfigured = true
```

Comment:
Do we really need `isConfigured` flag as we throw an error when there is no `key`?

Reply:
Yes, because the other hook functions are still being executed and would also throw not useful error messages